pom.xml

```diff
@@ -62,7 +62,7 @@
         <apache.curator.version>2.9.1</apache.curator.version>
         <jetty.version>9.2.5.v20141112</jetty.version>
         <jersey.version>1.19</jersey.version>
-        <druid.api.version>0.3.17</druid.api.version>
+        <druid.api.version>0.3.18</druid.api.version>
```

Comment:
there is a PR to merge druid-api 0.3.17 into druid - https://github.com/druid-io/druid/pull/2720